test/tests/toolbar.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+
+const { ariaTest } = require('..');
+const { By, Key } = require('selenium-webdriver');
+const assertAttributeValues = require('../util/assertAttributeValues');
+const assertAriaLabelledby = require('../util/assertAriaLabelledby');
+const assertAriaRoles = require('../util/assertAriaRoles');
+const assertRovingTabindex = require('../util/assertRovingTabindex');
+
+const exampleFile = 'toolbar/toolbar.html';
+
+const ex = {
+  toolbarSelector: '#ex1 [role="toolbar"]',
+  toolbarLabel: 'Example Toolbar',
+  toolSelector: '#ex1 [role="button"]',
+  menuSelector: '#ex1 button',
+  allToolSelectors: [
+    '#ex1 [role="button"]:nth-of-type(1)',
+    '#ex1 [role="button"]:nth-of-type(2)',
+    '#ex1 [role="button"]:nth-of-type(3)',
+    '#ex1 [role="button"]:nth-of-type(4)',
+    '#ex1 [role="button"]:nth-of-type(5)',
+    '#ex1 button'
+  ],
+  tabbaleItemBeforeToolbarSelector: '[href="../../#toolbar"]',
+  tabbaleItemAfterToolbarSelector: '[href="../../#kbd_roving_tabindex"]'
+};
+
+const clickAndWait = async function (t, selector) {
+  let element = await t.context.session.findElement(By.css(selector));
+  await element.click();
+
+  return await t.context.session.wait(
+    async function () {
+      let tabindex = await element.getAttribute('tabindex');
+      return tabindex === '0';
+    },
+    500,
+    'click did not set tabindex="0" on: ' + selector
+  ).catch((err) => { return err; });
+};
+
+const waitAndCheckFocus = async function (t, selector) {
+  return t.context.session.wait(
+    async function () {
```

Comment:
Does this need to be async? Everything in the function body is synchronous, unless `executeScript` is missing an `await`?